### PR TITLE
firewall: Show number of active zones on the network page

### LINKED ISSUE #12952: Do not state confusing number of rules in firewall
On networking page (`/network`) the description on firewall panel states `10 Active Rules`, however when I open the firewall page (`/network/firewall`) I see that I have two zones, first has 8 rules and the second one has 3 rules, so 10 rules seems very confusing (although it is correct, because one rule is shared between these two zones). Maybe we could show `11 rules in 2 zones` or `10 unique rules in 2 zones` or something similar.

### PR DESCRIPTION
**Pull request by Gundersanne**
image

### CODE REVIEW

File: pkg/networkmanager/interfaces.js [hunk 1]
```diff
@@ -2347,10 +2347,7 @@ PageNetworkInterface.prototype = {
             $('#networking-firewall').show();
             renderFirewallState();
 
-            var n = firewall.enabledServices.size;
-
-            /* HACK: use n.toString() here until cockpit.format() handles integer 0 args correctly */
-            var summary = cockpit.format(cockpit.ngettext("$0 Active Rule", "$0 Active Rules", n), n.toString());
+            const summary = cockpit.format(_("$0 Active Zones"), firewall.activeZones.size);
```
Comment (martinpitt) [inline pkg/networkmanager/interfaces.js]: Can it not happen that firewall.activeZones.size is zero? Or did we fix that cockpit.format() bug?

Comment (Gundersanne) [reply pkg/networkmanager/interfaces.js]: when I replaced it with 0 it seemed fine, but I guess I better actually check :p

Comment (Gundersanne) [reply pkg/networkmanager/interfaces.js]: it seems it got handled in #9062

Comment (marusak) [reply pkg/networkmanager/interfaces.js]: Should this use `ngettext`? Now it can be `1 Active Zones`, or `cockpit.format` can deal with it?

Comment (Gundersanne) [reply pkg/networkmanager/interfaces.js]: oh you're right, yea i'l ltag this for tomorrow.

Comment (martinpitt) [reply pkg/networkmanager/interfaces.js]: Argh, well spotted @marusak!

File: test/verify/check-networking-firewall [hunk 2]
```diff
@@ -56,21 +56,21 @@ class TestFirewall(NetworkCase):
...
-        active_rules = get_active_rules(m)
-        b.wait_in_text("#networking-firewall-summary", "{} Active Rule".format(len(set(active_rules))))
+        active_zones = len(m.execute("firewall-cmd --get-active-zones | grep '^[a-zA-Z]'").split())
+        b.wait_in_text("#networking-firewall-summary", "{} Active Zone".format(active_zones))
 
         self.toggle_onoff("#networking-firewall-switch")
         self.wait_onoff("#networking-firewall-switch", False)
         wait_unit_state(m, "firewalld", "inactive")
-        b.wait_in_text("#networking-firewall-summary", "0 Active Rules")
+        b.wait_in_text("#networking-firewall-summary", "0 Active Zone")
```
Comment (martinpitt) [inline test/verify/check-networking-firewall]: Your previous version with "0 Active Zones" was right -- in English (and many other languages), plural is being used for "0 things".

**Pull request merged**